docker.js

```diff
@@ -426,7 +426,14 @@ export const parseImageName = (fullImageName) => {
     return nameObj;
   }
   // ensure it's lowercased
-  fullImageName = fullImageName.toLowerCase();
+  fullImageName = fullImageName.toLowerCase().trim();
+
+  // Extract platform
+  if (fullImageName.startsWith("--platform=")) {
```

Comment:
Would it always start with the platform or could it be present at the end along with other options?

Reply:
According to the [docs](https://docs.docker.com/engine/reference/builder/#from), it should follow one of the following three structures

```
FROM [--platform=<platform>] <image> [AS <name>]
FROM [--platform=<platform>] <image>[:<tag>] [AS <name>]
[FROM](https://docs.docker.com/engine/reference/builder/#from) [--platform=<platform>] <image>[@<digest>] [AS <name>]

```